src/lib/arrays.js

```diff
@@ -0,0 +1,107 @@
+const getNthElement = (index, array) => {
+  if (index < array.length) {
+    return array[index];
+  }
+  return array[index - array.length];
+};
+
+const arrayToCSVString = array => {
+//  console.log(array.toString());
+  return array.toString();
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+function bob(bsr) {
+  return bsr;
+}
+
+const addToArray2 = (element, array) => {
+  const newArray = array.map(bob);
+  newArray.push(element);
+  console.log(newArray);
+  return newArray;
+};
+
+
+const removeNthElement = (index, array) => {
+if (index > -1) {
+  array.splice(index, 1);
+}
+return(array);
+}
+
+const numbersToStrings = numbers => {
+  const arrayToString = numbers.toString();
+  return arrayToString.split(',');
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(words => words.toUpperCase());
+};
+
+const reverseWordsInArray = strings => {
+  const reverse = strings.map(item => item.split('').reverse().join(''));
+  //console.log(reverse);
```

Comment:
This console.log could be deleted. And you could just return it directly instead of assigning it to a variable.